### ISSUE TITLE
Add the user name in authentication error message 

### ISSUE THREAD

**Issue by davidfestal** [fabric8-services/fabric8-auth#683]
Currently, when trying to login with a user that was not provisioned to OSIO, the following error is returned in the request body:
```javascript
{ errors: [
    error : {
        code : 'forbidden_error',
        detail : 'user is not authorized to access OpenShift'
    }
] }
```

It would be great if you could include the name of the user (RHD username) in this error message, like this:
```javascript
{ errors: [
    error : {
        code : 'forbidden_error',
        detail : 'user is not authorized to access OpenShift: dfestal'
    }
] }
```

That would allow a much better tracking of provisioning failures.

**Comment by sbose78** [comment]
@sbryzak , could you consider this in your refactoring? The idea is to ensure that the error message has the RHD username who the code finds out as 'not approved' ,ie, not present in the DB.

**Comment by sbryzak** [comment]
@sbose78 I'll take care of it!

**Comment by sbose78** [comment]
@sbryzak , Just clarifying :) We are looking to get this returned in the response body's message.

**Comment by davidfestal** [comment]
@sbose78 @sbryzak any idea when it will be available ?

**Comment by sbryzak** [comment]
@davidfestal We're currently working on getting the mother of all PR's merged to our master branch tonight, however if that goes well I'll have plenty of time to take a look at this tomorrow.

**Comment by davidfestal** [comment]
@sbose78 @sbryzak any news ?

**Comment by xcoulon** [comment]
the PR that @sbryzak was talking about is still under review: https://github.com/fabric8-services/fabric8-auth/pull/689

**Comment by davidfestal** [comment]
OK, thanks @xcoulon  for the information.

**Comment by sbryzak** [comment]
Just an update, I finally merged #689 this morning which was the blocker for this issue.  I'm now looking at this issue as my top priority, and I apologise for the lengthy delay in addressing it.
